docs/getting-started/keeping-backstage-updated.md

```diff
@@ -78,3 +78,15 @@ duplicate definitions:
 # Add --fix to attempt automatic resolution in yarn.lock
 yarn backstage-cli versions:check
 ```
+
+## Proxy
```

Comment:
@stephenglass, thoughts on this?

Reply:
Given that the `versions:bump` command is the only CLI command to use/need proxy config imo it makes more sense to keep it tied to the relevant [versions:bump command docs](https://backstage.io/docs/tooling/cli/commands#versionsbump) and the [Keeping Backstage Updated](https://backstage.io/docs/getting-started/keeping-backstage-updated/) docs rather than the CLI overview page. 

I did however push an update that makes a reference from the versions:bump command docs to the keeping backstage updated page as well as adding a note in the `help-im-behind-a-corporate-proxy.md` docs 😀